README.md

```diff
@@ -100,15 +100,16 @@ It can:
 
 In some cases, the vault's curator or allocators may want to keep some funds "idle", to guarantee lenders that some liquidity can be withdrawn from the vault (beyond the liquidity of each of the vault's markets).
 
-To achieve this, it is advised to allocate "idle" funds to any market on Morpho Blue having:
+To achieve this, it is advised to allocate "idle" funds to any market on Morpho Blue having this canonical configuration:
 
-- The vault's asset as loan token.
-- No collateral token (`address(0)`).
-- An IRM that does not trigger a revert (`address(0)`).
-- An arbitrary oracle (`address(0)`).
-- An arbitrary LLTV (`0`).
+- `loanToken`: The vault's asset to be able to supply/withdraw funds.
+- `collateralToken`: `address(0)` (not necessary since no funds will be borrowed on this market)
+- `irm`: `address(0)` (Morpho Blue will skip the call to the IRM in this case, thus reducing the gas cost)
+- `oracle`: `address(0)` (not necessary since no funds will be borrowed on this market)
+- `lltv`: `0` (not necessary since no funds will be borrowed on this market)
```

Comment:
I'm not sure that here it's clear that you NEED at least one of collat=0 and oracle=0 otherwise people might be able to borrow your assets